File: js/src/ForceLoad.js
/* eslint camelcase: off */
import { DOMWidgetModel } from '@jupyter-widgets/base';
import { version } from './version';

export class ForceLoadModel extends DOMWidgetModel {
    defaults() {
        return {
            ...super.defaults(),
            ...{
                _model_name: 'ForceLoadModel',
                _model_module: 'jupyter-vue',
                _model_module_version: version,
            },
        };
    }
}

ForceLoadModel.serializers = {
    ...DOMWidgetModel.serializers,
};

File: ipyvue/ForceLoad.py
from traitlets import Unicode
from ipywidgets import DOMWidget
from ._version import semver


class ForceLoad(DOMWidget):
    _model_name = Unicode("ForceLoadModel").tag(sync=True)
    _model_module = Unicode("jupyter-vue").tag(sync=True)
    _model_module_version = Unicode(semver).tag(sync=True)


force_load_instance = ForceLoad()
